p2panda-js/test/index.test.ts

```diff
@@ -15,4 +15,14 @@ describe('KeyPair', () => {
     const keyPairSecond = KeyPair.fromPrivateKey(keyPair.privateKey());
     expect(keyPair.publicKey()).to.eq(keyPairSecond.publicKey());
   });
+
+  it('signs and validates', async () => {
+    const { KeyPair } = await p2panda;
+    const keyPair = new KeyPair();
+    const message = 'hello panda';
+    const sig = keyPair.sign(message);
+    expect(keyPair.verify(message, sig)).to.be.undefined;
```

Comment:
Should we change the api to have `to.be.true` here? Testing for undefined feels very un-javascripty but I'm not sure how to have that and an acceptable rust `Result` as well.

Reply:
Agreed, this would be better. You can simply use `is_ok()` and it will return a boolean.

Reply:
You mean calling `is_ok()` on the return value? I don't know if that's possible?. I found another solution though!

Reply:
Your solution is better! It still keeps the actual error. 

Using `is_ok()` would work, you would *ignore* the error and convert it just into a boolean. For example like this:

```rust
fn do_something(&self) -> bool {
   self.im_doing_something_which_returns_a_result().is_ok()
}
```